core/contracts/financial_templates/Liquidatable.sol

```diff
@@ -387,6 +386,5 @@ contract Liquidatable is PricelessPositionManager {
         returns (LiquidationData storage liquidation)
     {
         liquidation = liquidations[sponsor][uuid];
-        require(liquidation.liquidator != address(0), "Liquidation does not exist: liquidator address is not set");
```

Comment:
`PricelessPositionManager` has a check in `_getPositionData` that the position is valid, which is why I included this line. Do you think its ok to just allow caller of `_getLiquidationData` to revert if it tries to perform operations on an invalid Liquidation?

The tradeoff might be more obscure revert messages.

Reply:
Ah, I made a bad assumption here -- that the only invalid liquidations were ones that were uninitialized (and therefore, past the end of the array). Once a liquidation is fully resolved, we delete it, so we'd want to protect users from accessing those as well. 

I'll add this revert back. Thanks for pointing this out. Definitely an oversight.

Reply:
Done.